units/en/unit1/quiz.mdx

```diff
@@ -0,0 +1,168 @@
+
+### Q1: What is Reinforcement Learning?
+
+<details>
+<summary>Solution</summary>
+
+Reinforcement learning is a **framework for solving control tasks (also called decision problems)** by building agents that learn from the environment by interacting with it through trial and error and **receiving rewards (positive or negative) as unique feedback**.
+
+</details>
+
+
+
+### Q2: Define the RL Loop
+
+<img src="https://huggingface.co/datasets/huggingface-deep-rl-course/course-images/resolve/main/en/unit1/rl-loop-ex.jpg" alt="Exercise RL Loop"/>
+
+At every step:
+- Our Agent receives ______ from the environment
+- Based on that ______ the Agent takes an ______
+- Our Agent will move to the right
+- The Environment goes to a ______
+- The Environment gives a ______ to the Agent
+
+
+<Question
+	choices={[
+		{
+			text: "an action a0, action a0, state s0, state s1, reward r1",
+			explain: "At every step: Our Agent receives **state s0** from the environment. Based on that **state s0** the Agent takes an **action a0**. Our Agent will move to the right. The Environment goes to a **new state s1**. The Environment gives **a reward r1** to the Agent."
+		},
+		{
+			text: "state s0, state s0, action a0, new state s1, reward r1",
+			explain: "",
+      correct: true
+		},
+		{
+			text: "a state s0, state s0, action a0, state s1, action a1",
+      explain: "At every step: Our Agent receives **state s0** from the environment. Based on that **state s0** the Agent takes an **action a0**. Our Agent will move to the right. The Environment goes to a **new state s1**. The Environment gives **a reward r1** to the Agent."
+		}
+	]}
+/>
+
+### Q3: What's the difference between a state and an observation?
+
+<Question
+	choices={[
+		{
+			text: "The state is a complete description of the state of the world (there is no hidden information)",
+			explain: "",
+      correct: true
+		},
+    {
+			text: "The state is a partial description of the state",
+			explain: ""
+		},
+    {
+      text: "The observation is a complete description of the state of the world (there is no hidden information)",
+      explain: ""
+    },
+    {
+      text: "The observation is a partial description of the state",
+      explain: "",
+      correct: true
+    },
+    {
+      text: "We receive a state when we play with chess environment",
+      explain: "Since we have access to the whole checkboard information.",
+      correct: true
+    },
+    {
+      text: "We receive an observation when we play with chess environment",
+      explain: "Since we have access to the whole checkboard information."
+    },
+    {
+      text: "We receive a state when we play with Super Mario Bros",
+      explain: "We only see a part of the level close to the player, so we receive an observation."
+    },
+    {
+      text: "We receive an observation when we play with chess environment",
+      explain: "We only see a part of the level close to the player.",
+      correct: true
```

Comment:
Not sure how well it will work with multiple correct answers

Reply:
I tested with preview and it works :partying_face: